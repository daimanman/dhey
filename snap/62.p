{"C":1,"Duration":229382719208,"EndTime":1537175082719130600,"Err":"","FileId":"3","InputFileName":"","Method":"POST","N":1,"P":null,"Payload":"","QH":{},"Remark":"压力测试","ReqNums":1,"StartTime":1537174853336411400,"Status":1,"TaskId":62,"Type":"FILE","Url":"http://127.0.0.1:9091/api/v1/oss","Z":""}