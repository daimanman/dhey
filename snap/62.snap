{"AvgTotal":229.3807653,"Fastest":229.3807653,"Slowest":229.3807653,"Average":229.3807653,"Rps":0.004359563447668033,"AvgConn":0,"AvgDNS":0,"AvgReq":0,"AvgRes":0.0010357,"AvgDelay":229.3797296,"ConnMax":0,"ConnMin":0,"DnsMax":0,"DnsMin":0,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0.0010357,"DelayMax":229.3797296,"DelayMin":229.3797296,"Lats":[229.3807653],"ConnLats":[0],"DnsLats":[0],"ReqLats":[0],"ResLats":[0.0010357],"DelayLats":[229.3797296],"Total":229380765300,"ErrorDist":{},"StatusCodeDist":{"400":1},"SizeTotal":15,"SizeReq":15,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":229.3807653,"Count":1,"Frequentcy":1},{"Mark":229.3807653,"Count":0,"Frequentcy":0},{"Mark":229.3807653,"Count":0,"Frequentcy":0},{"Mark":229.3807653,"Count":0,"Frequentcy":0},{"Mark":229.3807653,"Count":0,"Frequentcy":0},{"Mark":229.3807653,"Count":0,"Frequentcy":0},{"Mark":229.3807653,"Count":0,"Frequentcy":0},{"Mark":229.3807653,"Count":0,"Frequentcy":0},{"Mark":229.3807653,"Count":0,"Frequentcy":0},{"Mark":229.3807653,"Count":0,"Frequentcy":0},{"Mark":229.3807653,"Count":0,"Frequentcy":0}],"TaskId":62,"Duration":229.3807653}